python/MooseDocs/base/executioners.py

```diff
@@ -428,6 +431,7 @@ def execute(self, nodes, num_threads=1):
             p.start()
 
         for job in jobs:
+            self._updateAttributes(page_attributes) # run local updates one last time before finalizing
             job.join()
```

Comment:
@aeslaughter this is all I had to change

Reply:
Alright, I think I understand the reason. I will add a comment with my thoughts a bit later today.